x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/query/header/index.tsx

```diff
@@ -74,41 +90,69 @@ const QueryTabHeaderComponent: React.FC<Props> = ({
   const shouldShowQueryBuilder = isDataProviderVisible || timelineType === TimelineType.template;
 
   return (
-    <EuiFlexGroup gutterSize="s" direction="column">
-      <EuiFlexItem>
-        <StatefulSearchOrFilter filterManager={filterManager} timelineId={timelineId} />
-      </EuiFlexItem>
-      {showCallOutUnauthorizedMsg && (
-        <EuiFlexItem>
-          <EuiCallOut
-            data-test-subj="timelineCallOutUnauthorized"
-            title={i18n.CALL_OUT_UNAUTHORIZED_MSG}
-            color="warning"
-            iconType="warning"
-            size="s"
-          />
-        </EuiFlexItem>
-      )}
-      {status === TimelineStatus.immutable && (
-        <EuiFlexItem>
-          <EuiCallOut
-            data-test-subj="timelineImmutableCallOut"
-            title={i18n.CALL_OUT_IMMUTABLE}
-            color="primary"
-            iconType="warning"
-            size="s"
-          />
+    <StyledEuiFlyoutHeader data-test-subj={`${activeTab}-tab-flyout-header`} hasBorder={false}>
```

Comment:
@logeekal , not sure if there was a reason all of this didn't just live here before, but I moved everything for the header here

Reply:
@michaelolo24 , makes sense.. Thanks.